src/pages/medicines/[medicineId]/index.tsx

```diff
@@ -14,7 +14,7 @@ function MedicineInfo() {
       <div
         className={`
           md:hidden
-          text-m-headline1-b flex items-center justify-between text-gray-600 py-4 px-5 sticky top-0 bg-white shadow-[0px_2px_5px_0px_rgba(0,0,0,0.05)]`}>
+          text-m-headline1-b flex items-center justify-between text-gray-600 py-4 px-5 sticky top-[60px] mb-5 bg-white shadow-[0px_2px_5px_0px_rgba(0,0,0,0.05)] z-[50]`}>
```

Comment:
z-index 설정하는 부분에 값이 잘못 들어간 것 같아요 확인 부탁드립니다~
```suggestion
          text-m-headline1-b flex items-center justify-between text-gray-600 py-4 px-5 sticky top-[60px] mb-5 bg-white shadow-[0px_2px_5px_0px_rgba(0,0,0,0.05)] z-50`}>
```